docs/source/interfaces/Multi-dataset Fitting.rst

```diff
@@ -0,0 +1,138 @@
+
+Multi-dataset Fitting Interface
+===============================
+
+.. contents:: Table of Contents
+  :local:
+
+Interface Overview
+------------------
+
+The Multi-dataset fitting interface can be used to fit multiple spectra using
+the same model and starting parameters for each spectrum. This interface can be
+accessed from the main menu of MantidPlot, in *Interfaces → General →
+Multi-dataset Fitting*. The interface consists of three major parts: The fit
+property window, the spectrum & fit viewer, and the list of spectra to operate
+on. 
+
+.. figure::  ../images/multidataset-fitting/multidataset-fitting-labelled.png
+   :align: center
+   :width: 1000px
+
+   A high level overview of the various sections of the multidataset fitting
+   interface.
+
+List of Spectra
+---------------
+
+To start fitting data with this interface some data of interest first needs to
+be loaded into the view. Using the "Add workspace" button you can choose a
+workspace to add spectra from. This will let you choose from a list of loaded
+workspaces and optionally select which workspace indices to include for
+fitting. Spectra can be removed by highlighting a corresponding row (or rows)
+and clicking "Remove". To remove all spectra click the "workspace" column name
+and then click the "Remove" button.
+
+.. figure::  ../images/multidataset-fitting/multidataset-add-workspace.png
+   :align: center
+   :width: 800px
+
+   The dialog box shown after clicking "Add workspace".
+
+
+After a list of spectra have been imported from a workspace the table of
+spectra will be populated with one entry for each spectrum. Here you can see
+which workspace the spectrum came from, what the workspace index was, and what
+the start and end `x` values are for this particular spectrum for each fit. The
+start and end values are editable and can be changed by the user by double
+clicking on an entry.
+
+.. figure::  ../images/multidataset-fitting/multidataset-spectra.png
+   :align: center
+   :width: 800px
+
+   The list of spectra after loading a workspace.
+
+Fit Property Browser
+--------------------
+
+The fit property browser is located on the left of the interface. The operation
+of the fit property browser is very similar to the `general fitting view
+<https://www.mantidproject.org/MBC_Setup_And_Execute_First_Fit>`_ available in
+Mantid but with slightly reduced functionality. The fit property browser is
+split into two major parts. Th top panel allows the user to define the
+specifics of the model they wish to fit to the dataset. The bottom panel
+contains the parameters for the minimizer and general parameters not associated
+with the model (e.g. the number of iterations). The bottom panel also allows
+the user to choose between simultaneous and sequential fitting mode:
+
+- **Simultaneous Fitting**: Fit all spectra at the same time as a multi-domain
+  function.
+
+- **Sequential Fitting**: Fit the spectra one after another, using the
```

Comment:
Sequential fitting has two modes set via `Fit Type` property:
1. Sequential, in which the fitted parameters of the previous spectrum are used as the starting values for the next one.
2. Individual, when all spectra have the same starting values.

Reply:
Thanks! I didn't know we could operate in both modes.